lib/kernel.js

```diff
@@ -101,7 +101,10 @@ module.exports = function(config, cb) {
 
   function stop() {
```

Comment:
they are exactly the same thing. Older term was 'server', but service is much more appropriate because a service is not always a server (even though the first one was, and so the variable became `_servers`).